backend/app.py

```diff
@@ -0,0 +1,69 @@
+from http import server
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import exc
```

Comment:
nothing wrong with this but im just curious what is exc?

Reply:
it helps handle sqlalchemy errors